parser/ParseStatement.hpp

```diff
@@ -195,22 +197,54 @@ class ParseStatementCreateIndex : public ParseStatement {
      *                            If specified as null, then index is created
      *                            on all the attributes.
      * @param index_type The type of index to create.
-     * @param index_property_list A list of properties for the index that modify
-     *                            the default properties of the index.
      **/
     ParseStatementCreateIndex(const int line_number,
                               const int column_number,
                               ParseString *index_name,
                               ParseString *relation_name,
-                              PtrList<ParseString> *attribute_name_list,
+                              PtrList<ParseAttribute> *attribute_list,
+                              ParseString *index_type)
+      : ParseStatement(line_number, column_number),
+        index_name_(index_name),
+        relation_name_(relation_name),
+        attribute_list_(attribute_list),
+        index_type_(index_type) {
+      initializeIndexType();
+    }
+
+    /**
+     * @brief Constructor.
+     *
+     * @param index_name The name of the index to create.
+     * @param relation_name The name of the relation to create index upon.
+     * @param attribute_name_list A list of attributes of the relation
+     *                            on which the index has to be created.
+     *                            If specified as null, then index is created
+     *                            on all the attributes.
```

Comment:
These lines above should align with the first letter of `attribute_name_list` in Line 220.